app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class KeywordsController < ApplicationController
+  before_action :ensure_authentication
+
+  def new
+    render locals: {
+      keyword: Keyword.new
+    }
+  end
```

Comment:
since we have the CsvImportForm now, I think we should use it here too. How about:
```ruby
class KeywordsController
  before_action :set_csv_form
  
  def new
     render local: {
       csv_form: @csv_form
     }
  end

   def create
      if @csv_form.save(create_params)
      ...
      end
   end

  private
  
  def set_csv_form
     @csv_form = CsvImportForm.new(user: current_user)
  end

  def create_params
    params.require(:csv_form).permit(:file)
  end
end


class CsvImportForm
  ...
  include ActiveModel::Attributes

  attribute :user, :file

  def initialize(attributes = {})
    super(attributes)
  end

  def save(attributes)
    assign_attributes(attributes)
    
    return false unless valid?
    ....
  end
end
```

then in the view, you can render using the form instead of `keyword` model
```erb
# views/keywords/new.html.slim
= form_with(model: csv_form, local: true, url: keywords_path) do |form|
  h5.mb-3 = t('keyword.upload_keywords_csv_file')
  .form-group
    = form.file_field :file, class: "form-control-file", required: true, accept: 'text/csv'

  .action
    = form.submit t('keyword.upload'), class: "btn btn-dark"
```

Reply:
Fixed in ff07fd9a5746c051747bfb774e6a923819f36d1b